docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+web:
+  build: .
+  command: 'bash -c ''npm start'''
+  working_dir: /app/user
+  environment:
+    PORT: 8080
+  ports:
+    - '8080:8080'
```

Comment:
Nope, `REDIS_PORT` is defined in the redis container, when you link containers they are exposed to eachothers' environment variables.